src/main/java/paimon/tasklist/TaskList.java

```diff
@@ -0,0 +1,102 @@
+package paimon.tasklist;
+
+import java.util.ArrayList;
+
+import paimon.items.Todo;
+
+/**
+ * TaskList class that stores the Todo tasks. 
+ * Aslo can store Deadline and Event tasks. 
+ */
+public class TaskList implements Iterable<Todo> {
+    private ArrayList<Todo> items;
+
+    public TaskList() {
+        this.items = new ArrayList<>();
+    }
+
+    public Todo get(int index) {
+        return this.items.get(index);
+    }
+
+    /**
+     * Adds an item to the list
+     * 
+     * @param task
+     */
+    public void add(Todo task) {
+        this.items.add(task);
+    }
+
+    /**
+     * Removes an item from the list
+     * 
+     * @param index index of the item to be removed, 0 indexed
+     * @return the item that was removed
+     */
+    public Todo remove(int index) {
+        return this.items.remove(index);
+    }
+
+    /**
+     * Gets the number of items in the list
+     * 
+     * @return number of items in the list
+     */
+    public int size() {
+        return this.items.size();
+    }
+
+    /**
+     * Marks an item as done
+     * 
+     * @param index index of the item to be marked as done, 0 indexed
+     */
+    public void mark(int index) {
+        this.items.get(index).mark();
+    }
+
+    /**
+     * Unmarks an item as done
+     * 
+     * @param index index of the item to be unmarked, 0 indexed
+     */
+    public void unmark(int index) {
+        this.items.get(index).unmark();
+    }
+
+    /**
+     * Finds tasks that contain the keyword
+     * 
+     * @param keyword the keyword to search for
+     * @return a TaskList only containing the tasks that contain the keyword
+     */
+    public TaskList find(String keyword) {
+        TaskList found = new TaskList();
+        for (Todo task : this.items) {
+            if (task.getDescription().contains(keyword)) {
+                found.add(task);
+            }
+        }
+        return found;
+    }
+
+    /**
+     * Lists all the current items in the list and outputs to screen. 
+     */
+    public void list_items() {
```

Comment:
Should list_items() be in snake case or camelCase?
```suggestion
    public void listItems() {
```